x-pack/legacy/plugins/lens/public/editor_frame_plugin/editor_frame/frame_layout.tsx

```diff
@@ -18,15 +18,16 @@ export interface FrameLayoutProps {
 export function FrameLayout(props: FrameLayoutProps) {
   return (
     <RootDragDropProvider>
-      <EuiFlexGroup>
-        {/* TODO style this and add workspace prop and loading flags */}
-        <EuiFlexItem grow={null}>{props.dataPanel}</EuiFlexItem>
-        <EuiFlexItem grow={5}>{props.workspacePanel}</EuiFlexItem>
-        <EuiFlexItem grow={null}>
+      <EuiPage className="lnsPage">
+        <EuiPageSideBar className="lnsSidebar">{props.dataPanel}</EuiPageSideBar>
+        <EuiPageBody className="lnsPageBody" restrictWidth={false}>
+          {props.workspacePanel}
+        </EuiPageBody>
+        <EuiPageSideBar className="lnsSidebar--right">
```

Comment:
With BEM styling you're expected to use the block and modifier together, as in `className="lnsSidebar lnsSidebar--right"`

Reply:
You are right, corrected this.